src/Custom/Responses/Tools/ComputerTool.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace OpenAI.Responses;
+
+[CodeGenType("ComputerUsePreviewTool")]
```

Comment:
nit: for the added CodeGenType attributes can we add the typical comment about what was customized?